components/automate-ui/src/app/modules/infra-proxy/data-bags-details/data-bags-details.component.html

```diff
@@ -70,7 +70,7 @@
             </ul>
           </div>
           <app-page-picker
-            class="data-bag-items-list-paging"
+            class="item-list-paging"
```

Comment:
Nice catch. 🎉 
However, there are a few more of the same--CSS classes being used that are never defined.
These couple are in your area, so please correct also:
https://github.com/chef/automate/blob/5f0d283159e37a6d0c98fcce4eef2a9a5c6ce2fb/components/automate-ui/src/app/modules/infra-proxy/clients/clients.component.html#L30
https://github.com/chef/automate/blob/68d5586caa32d8fa3a661b6456353f3e3f7db8b6/components/automate-ui/src/app/modules/infra-proxy/environments/environments.component.html#L39

These last ones are in different code areas, but your familiarity here might allow you to easily patch them, too; okay if not.
https://github.com/chef/automate/blob/bebe636f4016dc6b9e8451458476d9ecaa53cc1e/components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html#L125
https://github.com/chef/automate/blob/f8ca770fc9c7915a9658855d5130811e5875eae1/components/automate-ui/src/app/pages/integrations/detail/integrations-detail.component.html#L74

Reply:
@msorens I have added these changes in my last commit but Idk why it's not showing outdated as I checked my files the is there for the above undefined-CSS class.